.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+What?
```

Comment:
What about using Markdown sections?

```md
## What?
> Fill out a short description about the purpose of the PR.

...
```

Reply:
For example?

Reply:
I mean that the template is flat and it's hard to recognize the  sections. Because it's a Markdown document, why not using sections?

```mkd
## What?
## Why?
## How?
```

Reply:
Oh yes! That is the idea...that is what I have been using...
https://github.com/android10/Android-CleanArchitecture-Kotlin/blob/main/.github/pull_request_template.md

But for some reason, copy/paste did not work properly here :). Fixing